src/System.Private.CoreLib/shared/System/Collections/Generic/IDictionary.cs

```diff
@@ -45,6 +46,6 @@ ICollection<TValue> Values
         //
         bool Remove(TKey key);
 
-        bool TryGetValue(TKey key, out TValue value); // TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/26761
+        bool TryGetValue(TKey key, [MaybeNullWhen(false)] out TValue value);
```

Comment:
Github won't let me comment but I also think the constraint `where TKey : object` should be annotated as `where TKey : nonnull` I think roslyn was planning on delivering that as part of the attributes work. @jaredpar @MadsTorgersen correct me if I'm wrong.

Reply:
That's correct, but then this also wouldn't compile.  We can search/replace all of those when we have a compiler that recognizes nonnull.  These attributes are just attributes and thus don't actually require the compiler in order to compile.